azext_iot/operations/events3/_events.py

```diff
@@ -47,20 +66,24 @@ def executor(target, consumer_group, enqueued_time, properties=None,
     try:
         device_filter_txt = None
         if device_id:
-            device_filter_txt = ' filtering on device: {},'.format(device_id)
+            device_filter_txt = " filtering on device: {},".format(device_id)
 
         def stop_and_suppress_eloop():
             try:
                 loop.stop()
-            except Exception:  # pylint: disable=broad-except
+            except Exception:
                 pass
 
-        six.print_('Starting {}event monitor,{} use ctrl-c to stop...'.format('PnP ' if pnp_context else '',
-                                                                              device_filter_txt if device_filter_txt else ''))
+        six.print_(
+            "Starting {}event monitor,{} use ctrl-c to stop...".format(
```

Comment:
the spacing here is a bit strange to me.
I'd change it to "Starting {} event monitor,
where {} would be "PnP" instead of "PnP "